src/ui/Services/CourseMapper.cs

```diff
@@ -38,6 +38,12 @@ public SearchAndCompare.Domain.Models.Course MapToSearchAndCompareCourse(ApiClie
             // todo refactor out from Extension Method
             var routeName = new CourseVariantViewModel { Route = ucasCourseData?.ProgramType ?? "" }.GetRoute();
             var isSalaried = string.Equals(ucasCourseData?.ProgramType, "ss", StringComparison.InvariantCultureIgnoreCase);
+            var fees = courseEnrichmentModel.FeeUkEu.HasValue ? new Fees
+            {
+                Uk = (int) (courseEnrichmentModel.FeeUkEu ?? 0),
+                    Eu = (int) (courseEnrichmentModel.FeeUkEu ?? 0),
+                    International = (int) (courseEnrichmentModel.FeeInternational ?? 0),
+            } : null;
```

Comment:
would there ever be something with a FeeInternational but no FeeUkEu?

Reply:
this might be easier to read if it wasn't all in one statement, not sure though

Reply:
No Idea about fee uk vs eu vs international

but there another story to cover refactoring

https://trello.com/c/Mrj53nGo/199-course-preview-should-use-last-published-orgenrichment-not-current-orgenrichment-draft